Wikipedia/Code/TalkPageTopicReplyOnboardingView.swift

```diff
@@ -7,7 +7,16 @@ struct TalkPageTopicReplyOnboardingView: View {
 
     enum LocalizedStrings {
         static let title = WMFLocalizedString("talk-pages-topic-reply-onboarding-title", value: "Talk pages", comment: "Title of user education onboarding view for user and article talk pages.")
-        static let body = WMFLocalizedString("talk-pages-topic-reply-onboarding-body", value: "Talk pages are where people discuss how to make content on Wikipedia the best that it can be. Add a new discussion topic to connect and collaborate with a community of Wikipedians.\n\nPlease be kind, we are all humans here.", comment: "Body text for user education onboarding view for user and article talk pages. Please do not translate the \n\n characters as these indicate where to insert new lines.")
+        static let body = WMFLocalizedString("talk-pages-topic-reply-onboarding-body", value: "Talk pages are where people discuss how to make content on Wikipedia the best that it can be. Add a new discussion topic to connect and collaborate with a community of Wikipedians.\n\nPlease be kind, we are all humans here.", comment: "Body text for user education onboarding view for user and article talk pages.")
+        @available(iOS 15, *)
+        static var bodyiOS15: AttributedString? = {
+            let localizedString = WMFLocalizedString("talk-pages-topic-reply-onboarding-body-ios15", value: "Talk pages are where %1$@people discuss how to make content on Wikipedia the best that it can be.%1$@ Add a new discussion topic to connect and collaborate with a community of Wikipedians.\n\nPlease be kind, we are all humans here.", comment: "Body text for user education onboarding view for user and article talk pages. Parameters:\n* %1$@ - app-specific non-text formatting")
+            let attributedString = String.localizedStringWithFormat(
+                localizedString,
+                "**"
+            )
+            return try? AttributedString(markdown: attributedString)
```

Comment:
It's odd but now I'm seeing that the newline characters have no effect:

![Screen Shot 2022-11-04 at 9 13 07 AM](https://user-images.githubusercontent.com/3620196/200002046-2c0e3b49-d151-4b4d-9a2e-790d20dbb877.png)

Maybe we'll need to break this off into a separate text element for > iOS 15?

Reply:
I separated it into another string, I guess the markdown formatting is not compatible with the newline characters

Reply:
Looks great!